packages/synapse-react-client/src/components/widgets/facet-nav/FacetNav.tsx

```diff
@@ -203,8 +203,10 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
             <div className="FacetNav__row" role="list">
               {facets.map(facet => (
                 <div
-                  className="col-sm-12 col-md-4"
```

Comment:
These bootstrap grid styles were effectively only adding a padding left and right (because the responsive behavior is due to the display flex wrap styles!)